src/main/java/com/joseeduardo/todoist/controller/TarefaController.java

```diff
@@ -25,17 +32,46 @@ public TarefaController(TarefaService tarefaService) {
 		this.tarefaService = tarefaService;
 	}
 
-	@PostMapping("/{id}")
+	@PostMapping("/criar/{id}")
 	public ResponseEntity<Void> criar(@RequestBody CriarTarefaEntradaDTO criarTarefaEntradaDTO,
 			@PathVariable("id") Long id) {
 		tarefaService.criar(criarTarefaEntradaDTO, id);
 		return ResponseEntity.status(HttpStatus.CREATED).build();
 	}
 
-	@DeleteMapping("/{id}")
-	public ResponseEntity<Void> exlcuirPorID(@PathVariable("id") Long id) {
+	@DeleteMapping("/deletar/{id}")
+	public ResponseEntity<Void> exlcuir(@PathVariable("id") Long id) {
 		tarefaService.excluir(id);
 		return ResponseEntity.status(HttpStatus.OK).build();
 	}
-	
+
+	@GetMapping
+	public ResponseEntity<List<CriarTarefaEntradaDTO>> buscarTodos() {
+		List<TarefaEntity> tarefas = tarefaService.buscarTodos();
+		List<CriarTarefaEntradaDTO> tarefasEntradaDTO = tarefas.stream().map(x -> new CriarTarefaEntradaDTO(x))
+				.collect(Collectors.toList());
+		return ResponseEntity.ok().body(tarefasEntradaDTO);
+	}
+
+	@GetMapping("/status/{status}")
```

Comment:
```suggestion
	@GetMapping("/{id}")
```

é passado um id, certo? ou é passado um status?